crawler/cv/spiders/huxiu.py

```diff
@@ -1,13 +1,55 @@
 # -*- coding: utf-8 -*-
 import scrapy
+from cv.items.article import ArticleItem
+import datetime
 
 
 class HuxiuSpider(scrapy.Spider):
     name = "huxiu"
     allowed_domains = ["huxiu.com"]
     start_urls = (
-        'http://www.huxiu.com/',
+        #'http://www.huxiu.com/',
+        'http://www.huxiu.com/article/146942',
     )
+    #custom_settings = {
+    #    'ITEM_PIPELINES': {
+    #        'cv.pipelines.a36kr.ArticlePipeline': 300
+    #    }
+    #}
 
     def parse(self, response):
-        pass
+        print 'start'.center(100, '=')
+        item = self.parse_page(response)
+        print item['created_ts']
```

Comment:
please remove these print, we'll replace it with python logging system, and you can debug with scrapy shell